spec/models/manageiq/providers/openstack/cloud_manager/refresher_spec.rb

```diff
@@ -14,92 +14,129 @@
 
     credentials = {:userid => username, :password => password}
     @ems.authentications << FactoryBot.create(:authentication, credentials)
+    @inventory = nil
+  end
+
+  def reset_cache
+    @ems.reset_openstack_handle
+
+    require "fog/openstack"
+    Fog::OpenStack.instance_variable_set(:@version, nil)
+  end
+
+  def refresh_inventory
+    with_vcr do
```

Comment:
We're going to need to take a parameter to pass in as the suffix since the different tests will need different VCRs (e.g. the admin account "fast" refresh will make different API calls and will need its own VCR cassette.

```suggestion
  def refresh_inventory(vcr_suffix = nil)
    with_vcr(vcr_suffix) do
```